src/github.com/matrix-org/dendrite/federationapi/routing/events.go

```diff
@@ -29,10 +27,7 @@ import (
 func GetEvent(
 	ctx context.Context,
 	request *gomatrixserverlib.FederationRequest,
-	_ config.Dendrite,
```

Comment:
Can anyone tell me why it was useful to have a function take in parameters that were never used?

Reply:
These were expected to be used to verify http request (around August 17).
But then in October or so everything was ported to common.MakeFedAPI which did the verification.
As a result some these parameters became redundant.